hack/make/release-deb

```diff
@@ -35,6 +35,51 @@ if [[ ! -f "$APTDIR/conf/distributions" ]]; then
 	done > "$APTDIR/conf/distributions"
 fi
 
+# create/update distributions file
+if [[ ! -f "$APTDIR/conf/apt-ftparchive.conf" ]]; then
+	cat <<-EOF > "$APTDIR/conf/apt-ftparchive.conf"
+	Dir {
+		ArchiveDir "${APTDIR}";
+		CacheDir "${APTDIR}/db";
+	};
+
+	Default {
+		Packages::Compress ". gzip bzip2";
+		Sources::Compress ". gzip bzip2";
+		Contents::Compress ". gzip bzip2";
+	};
+
+	TreeDefault {
+		BinCacheDB "packages-\$(SECTION)-\$(ARCH).db";
+		Directory "pool/\$(SECTION)";
+		Packages "\$(DIST)/\$(SECTION)/binary-\$(ARCH)/Packages";
+		SrcDirectory "pool/\$(SECTION)";
+		Sources "\$(DIST)/\$(SECTION)/source/Sources";
+		Contents "\$(DIST)/\$(SECTION)/Contents-\$(ARCH)";
+		FileList "$APTDIR/\$(DIST)/\$(SECTION)/filelist";
+	};
+	EOF
+
+	for suite in $(exec contrib/reprepro/suites.sh); do
+		cat <<-EOF
+		Tree "dists/${suite}" {
+			Sections "main testing experimental";
+			Architectures "i386 amd64";
+		}
+
+		EOF
+	done >> "$APTDIR/conf/apt-ftparchive.conf"
+fi
+
+if [[ ! -f "$APTDIR/conf/docker-engine-release.conf" ]]; then
```

Comment:
i will change them allllll because now its going to bother me its
inconsistent w the others

On Wed, Sep 2, 2015 at 9:21 AM, Tianon Gravi <notifications@github.com>
wrote:

> In hack/make/release-deb
> <https://github.com/docker/docker/pull/16001#discussion_r38552583>:
>
> > +		FileList "$APTDIR/\$(DIST)/\$(SECTION)/filelist";
> > +	};
> > +	EOF
> > +
> > +	for suite in $(exec contrib/reprepro/suites.sh); do
> > +		cat <<-EOF
> > +		Tree "dists/${suite}" {
> > +			Sections "main testing experimental";
> > +			Architectures "i386 amd64";
> > +		}
> > +
> > +		EOF
> > +	done >> "$APTDIR/conf/apt-ftparchive.conf"
> > +fi
> > +
> > +if [[ ! -f "$APTDIR/conf/docker-engine-release.conf" ]]; then
>
> heh, he's right that we've used "avoid [[ when not necessary" throughout
> most of the scripts in the repo, but I think you're right that this script
> doesn't, so oh well -- self-consistency FTW [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/16001/files#r38552583>.
>